R/utilities-qualification.R

```diff
@@ -336,30 +336,28 @@ getPlotSettings <- function(plotSettingsFromConfigurationPlot) {
 #' @param pkSimPortableFolder Folder where PK-Sim is located.
 #' If not specified, installation path will be read from the registry (available only in case of full **non-portable** installation).
 #' This option is **MANDATORY** for the portable version of PK-Sim.
-#' @param configurationPlanFile Name of the configuration plan to be generated. Default is 'report-configuration-plan.json'
+#' @param configurationPlanName Name of the configuration plan to be generated. Default is `"report-configuration-plan"`
 #' @param overwrite Logical defining if the contents of the output folder will be deleted, even if it not empty. Default is false.
 #' @param logFile Full path of log file where log output will be written. A log file will not be created if this value is not provided.
 #' @param logLevel Log verbosity (Debug, Information, Warning, Error). Default is Information.
 #' @param version Logical defining if version information is displayed
 #' @export
-startQualificationRunner <- function(
-                                     qualificationRunnerFolder,
+startQualificationRunner <- function(qualificationRunnerFolder,
                                      qualificationPlanFile,
                                      outputFolder,
                                      pkSimPortableFolder = NULL,
-                                     configurationPlanFile = NULL,
+                                     configurationPlanName = NULL,
                                      overwrite = TRUE,
                                      logFile = NULL,
                                      logLevel = NULL,
                                      displayVersion = FALSE) {
   validateIsFileExtension(qualificationPlanFile, "json")
   validateIsLogical(overwrite)
   validateIsLogical(displayVersion)
-  validateIsFileExtension(configurationPlanFile, "json", nullAllowed = TRUE)
-
+  
   options <- c(
     ifnotnull(pkSimPortableFolder, paste0("-p ", pkSimPortableFolder)),
-    ifnotnull(configurationPlanFile, paste0('-n "', configurationPlanFile, '"')),
+    ifnotnull(configurationPlanFile, paste0('-n "', configurationPlanName, '"')),

```

Comment:
@pchelle 
should this be `ifnotnull(configurationPlanName, paste0('-n "', configurationPlanName, '"')),`

?